monosketch-svelte/src/lib/mono/keycommand/interface.ts

```diff
@@ -22,6 +22,58 @@ export class Key {
     static readonly Z = 90;
 }
 
+export function getKeyFromEvent(e: KeyboardEvent): number {
```

Comment:
This is not good for maintenance. For example, when we add new support key, we have to list the key twice.

One option is changing `getCommandByKey` to accept the key value.

The other is combine `key` and `keyCode` into the same record

```
{
    key: 'Escape',
    code: 27
}
```